src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -52,23 +52,30 @@
   <!-- This project references the SonarLint TeamExplorer assembly which is VS-version specific -->
   <Import Project="..\SonarLint.VSSpecificAssemblies.props" />
 
-  <!-- Need to reference the Sonar Roslyn analyzers: we need to reference one of the assmblies,
-       and embed all of them in the VSIX. -->
+  <!-- ********************************************************** -->
+  <!-- Sonar C#/VB analyzers -->
+  <!-- ********************************************************** -->
   <Import Project="..\EmbeddedSonarAnalyzer.props" />
   <ItemGroup Label="Embedded SonarLint analyzers">
-    <!-- Reference the packages we want to import, but we don't want to include them as analyzers for project -->
+    <!-- Reference the packages we want to import, but don't include them as analyzers for this project -->
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
     <PackageReference Include="SonarAnalyzer.VisualBasic" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
 
     <!-- Select all of the analyzer binaries (we assume that the package restore has happened so the files are already on disc) -->
     <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
     <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.VisualBasic\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
 
-    <Reference Include="@(SonarAnalyzerAssemblies)" />
-    <!-- need to reference the common SonarAnalyzer.dll assembly directly -> no harm in referencing them all -->
+    <!-- We need to reference the core analyzer assembly since we use one of the types. -->
+    <Reference Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\SonarAnalyzer.dll" />
```

Comment:
I suppose this is an optimization versus the previous code?
I thought it was clearer as simply `Reference Include="@(SonarAnalyzerAssemblies)"`